packages/modular-scripts/src/__tests__/esmView.test.ts

```diff
@@ -85,6 +90,8 @@ describe('modular working with an esm-view', () => {
         },
         tempModularRepo,
       );
+      // Wait two seconds to ensure app started fully
+      await new Promise((f) => setTimeout(f, 2000));
```

Comment:
Not an impressive solution but I haven't seen the problem appear since I've added it

Could be shortened to `await setTimeout(2000)` once we drop node 14